File: introduction/ITP1_2_C-SortingThreeNumbers.java
import java.util.*;
import java.util.stream.Collectors;

public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String[] line = sc.nextLine().split(" ");

    int[] arr = Arrays.stream(line).mapToInt(Integer::parseInt).toArray();
    Arrays.sort(arr);

    System.out.println(Arrays.stream(arr).mapToObj(Integer::toString).collect(Collectors.joining(" ")));
  }
}

File: introduction/ITP1_2_C-Sorting Three Numbers.c
#include <stdio.h>

sort(int *a, int *b) {
  return *a - *b;
}

int main(void) {
  int arr[3];
  scanf("%d %d %d", &arr[0], &arr[1], &arr[2]);
  qsort(arr, 3, 4, sort);
  printf("%d %d %d\n", arr[0], arr[1], arr[2]);
  return 0;
}
